src/redux/auth/authActions.js

```diff
@@ -0,0 +1,42 @@
+import { createAction } from '@reduxjs/toolkit'
+
+const registerRequest = createAction('auth/registerRequest')
+const registerSuccess = createAction('auth/registerSuccess')
+const registerError = createAction('auth/registerError')
+
+const loginRequest = createAction('auth/loginRequest')
+const loginSuccess = createAction('auth/loginSuccess')
+const loginError = createAction('auth/loginError')
+
+const googleRequest = createAction('auth/googleRequest')
```

Comment:
What for do you need this action?

Reply:
I add this for future logining when we take token from backend and save it in local storage